src/deep-asset/test/Asset.js

```diff
@@ -4,130 +4,82 @@ import chai from 'chai';
 import sinon from 'sinon';
 import sinonChai from 'sinon-chai';
 import {Asset} from '../lib.compiled/Asset';
+import {Instance} from '../node_modules/deep-kernel/lib.compiled/Microservice/Instance';
 import Kernel from 'deep-kernel';
-import Instance from '../node_modules/deep-kernel/lib.compiled/Microservice/Instance';
+import KernelFactory from './common/KernelFactory';
 
 chai.use(sinonChai);
 
 suite('Asset', function() {
-  let asset = new Asset();
+  let assetService = null;
+  let backendKernelInstance = null;
+  let frontendKernelInstance = null;
 
   test('Class Asset exists in Asset', function() {
     chai.expect(typeof Asset).to.equal('function');
   });
 
-  test('Check boot() method  for !kernel.isFrontend', function() {
-    let kernel = {isFrontend: false};
-    let spyCallback = sinon.spy();
-    asset.boot(kernel, spyCallback);
-    chai.expect(spyCallback).to.have.been.calledWith();
+  test('Load Kernels by using Kernel.load()', function(done) {
+    let error = null;
+
+    let callback = (frontendKernel, backendKernel) => {
+      chai.expect(error).to.be.eql(null);
+      chai.expect(backendKernel).to.be.not.eql({});
```

Comment:
- check if backendKernel is an instance of Kernel class instead of chai.expect(backendKernel).to.be.not.eql({});